nec_tables.py

```diff
@@ -438,4 +438,200 @@
+                        0.50:0.314,
+                        0.75:0.549,
+                        1.00:0.887,
+                        1.25:1.526,
+                        1.50:2.071,
+                        2.00:3.408,
+                        2.50:4.866,
+                        3.00:7.499,
+                        3.50:10.010,
+                        4.00:12.882,
+                        5.00:20.212,
+                        6.00:29.158},
+            # Article 352 - Rigid PVC Conduit (PVC), Schedule 80
+                 'PVC80':{
+                        0.50:0.217,
+                        0.75:0.409,
+                        1.00:0.688,
+                        1.25:1.237,
+                        1.50:1.711,
+                        2.00:2.874,
+                        2.50:4.119,
+                        3.00:6.442,
+                        3.50:8.688,
+                        4.00:11.258,
+                        5.00:17.855,
+                        6.00:25.598},
+            # Article 352 - Rigid PVC Conduit (PVC), Schedule 40
+                 'PVC40':{
+                        0.50:0.285,
+                        0.75:0.508,
+                        1.00:0.832,
+                        1.25:1.453,
+                        1.50:1.986,
+                        2.00:3.291,
+                        2.50:4.695,
+                        3.00:7.268,
+                        3.50:9.737,
+                        4.00:12.554,
+                        5.00:19.761,
+                        6.00:28.567},
+            # Article 353 - High-Density Polyethylene (HDPE)
+                 'HDPE':{
+                        0.50:0.285,
+                        0.75:0.508,
+                        1.00:0.832,
+                        1.25:1.453,
+                        1.50:1.986,
+                        2.00:3.291,
+                        2.50:4.695,
+                        3.00:7.268,
+                        3.50:9.737,
+                        4.00:12.554,
+                        5.00:19.761,
+                        6.00:28.567},
+            # Article 352 - Rigid PVC Conduit, Type A
+                 'PVC A':{
+                        0.50:0.385,
+                        0.75:0.650,
+                        1.00:1.085,
+                        1.25:1.767,
+                        1.50:2.324,
+                        2.00:3.647,
+                        2.50:5.453,
+                        3.00:8.194,
+                        3.50:10.694,
+                        4.00:13.723},
+            # Article 352 - Rigid PVC Conduit, Type EB
+                 'PVC EB':{
+                        2.00:3.874,
+                        3.00:8.709,
+                        3.50:11.365,
+                        4.00:14.448,
+                        5.00:22.195,
+                        6.00:31.530}}
+
+# Conduit material for AC resistance values
+# (XX,YY)  :: XX=conduit_type, YY=conduit_material
+conduit_material = [('EMT','Aluminum'),
```

Comment:
I think this might be better as a dictionary than a list of tuples.

Reply:
Converted to dictionary.